stumpy/floss.py

```diff
@@ -581,7 +581,7 @@ def update(self, t):
 
         D[zone_start:] = np.inf
 
-        T_subseq_isfinite = core.rolling_isfinite(self._T_isfinite, self._m)
+        T_subseq_isfinite = core.rolling_isfinite(self._T, self._m)
```

Comment:
Thanks to your code coverage, I realized that the `core.rolling_isfinite` accepts boolean array as well ! (see `core._rolling_isfinite_1d`)

So, everything was fine in the first place!

**side note:**
The description of the output of `core.rolling_isfinite` needs to be corrected in the docstring.

https://github.com/TDAmeritrade/stumpy/blob/c06f0e998d28e7b717bab1e4e32a2aeb897ddfc3/stumpy/core.py#L2297-L2298

Reply:
Ahh, you know what? I mis-read `self._T_isfinite` as `self._T_subseq_isfinite`. The former is for `isfinite` for each element of `T` while the latter is the `isfinite` for each subsequence of `T`. Now that makes much more sense why it never failed.

Reply:
> The description of the output of core.rolling_isfinite needs to be corrected in the docstring.

Sounds good. It was likely copied over from `rolling_nanmax`

Reply:
@seanlaw 
Feel free to close the PR. I will address it in another PR.